docs/API.md

```diff
@@ -87,7 +87,8 @@ A function used to convert a query string into an object that gets passed to rou
 While the router is matching, errors may bubble up, here is your opportunity to catch and deal with them. Typically these will come from async features like [`route.getComponents`](#getcomponentslocation-callback), [`route.getIndexRoute`](#getindexroutelocation-callback), and [`route.getChildRoutes`](#getchildrouteslocation-callback).
 
 ##### `onUpdate()`
-Called whenever the router updates its state in response to URL changes.
+Called whenever the router updates its state in response to URL changes. 
+Within this function the current location is accessible through `this.state.location.pathname`.
```

Comment:
ahh good point.

Also I want to be sure this would be the best place to run things on route changes (like the old onChange stuff)... saw some discussion on other parts of the repo and am now not really sure where I should be doing this stuff